not_so_basic_grunt.py

```diff
@@ -0,0 +1,100 @@
+import pygame
+import math
+
+from gameobject import GameObject
+
+
+class NotSoBasicGrunt(GameObject):
+    
+
+    def __init__(self, starting_position, universe):
+        self.position_ = starting_position
+
+        self.ID_ = self.create_ID()
```

Comment:
I made a change in another pull request so to make `ID_` lower case everywhere, so `ID_` becomes `id_`. I'd recommend changing it now before facing a mysterious bug later.